src/oss/confluent.sh

```diff
@@ -1674,11 +1713,14 @@ These are the available commands:
 
     acl         Specify acl for a service.
     config      Configure a connector.
+    consume     Use the kafkacat utility to consume from topics
     current     Get the path of the data and logs of the services managed by the current confluent run.
     destroy     Delete the data and logs of the current confluent run.
+    kafkacat    Use the kafkacat utility to produce to and consume from topics
```

Comment:
For my understanding, why do we expose this separately?

Reply:
There is some use for a generic `confluent kafkacat`  mode to provide access to the metadata listing mode (`-L`), group consumer (`-G`), query tools (`-Q`), etc.